docs/references/configuration_options_reference.md

```diff
@@ -293,6 +295,10 @@ String used to define the configuration of the broker if one is autogenerated. S
 - `--timeout=` milliseconds to wait for all the federates to connect to the broker (can also be entered as a time like '10s' or '45ms')
 - `--network_timeout=` - Time to establish a socket connection in ms. Times can also be entered as strings such as "15s" or "75ms".
 - `--error_timeout=` - Time in ms to wait after an error state is reached before terminating. Times can also be entered as strings such as "15s" or "75ms".
+- `--query_timeout=` - Time in ms to wait for a query to complete. Times can also be entered as strings such as "15s" or "75ms".
+- `--children=` - The minimum number of child objects the broker should expect before allowing entry to the initializing state.
+- `--subbrokers=` - The minimum number of child objects the broker should expect before allowing entry to the initializing state. Same as `--children` but might be clearer in some cases with multilevel hierarchies.
+- `--brokerkey=` - A broker key to use for connections
```

Comment:
Can we add more help details on `brokerkey`?  Could we add "... to ensure federates are connecting with a specific broker."?